pkg/cloud_provider/file/file.go

```diff
@@ -153,6 +155,11 @@ func (manager *gcfsServiceManager) CreateInstance(ctx context.Context, obj *Serv
 }
 
 func (manager *gcfsServiceManager) CreateInstanceFromBackupSource(ctx context.Context, obj *ServiceInstance, sourceSnapshotId string) (*ServiceInstance, error) {
+	// enterprise tier Filestore does not yet support Backup
```

Comment:
we can fail early in the CreateVolume call itself. before the GET call of backups. https://screenshot.googleplex.com/3cH9tGAWwkwArpB